app/src/org/commcare/activities/LoginActivity.java

```diff
@@ -532,35 +613,65 @@ protected void restoreEnteredTextFromRotation() {
         }
     }
 
-    protected void populateAppSpinner(ArrayList<ApplicationRecord> readyApps) {
+    protected boolean populateAppSpinner(ArrayList<ApplicationRecord> readyApps) {
         ArrayList<String> appNames = new ArrayList<>();
+
         appIdDropdownList.clear();
+
+        boolean includeDefault = ConnectIDManager.isConnectIDIntroduced() && !ConnectIDManager.isSignedIn();
+        if(includeDefault) {
+            appNames.add(Localization.get("login.app.direct"));
+            appIdDropdownList.add("");
+        }

```

Comment:
The default that this code refers to is an additional option that says "Login to your app directly" and server as a placeholder when we're trying to encourage the user to unlock ConnectID instead of proceeding directly to their traditional credentials. When ConnectID is not introduced or is already unlocked, the selector is set to the currently seated app. Choosing the selector value happens a few lines later, around lines 640-648.